cypress/support/pages/cartPageObject.js

```diff
@@ -0,0 +1,38 @@
+import PageObject from '../PageObject';
+
+class CartPage extends PageObject {
+  visitCart() {
+    cy.contains('#cartur', 'Cart').click();
+  };
+
+  assertProductInCart(productName) {
+    cy.contains(productName).should('be.visible');
+  };
+
+  placeOrderBtn() {
+    cy.contains('.btn-success', 'Place Order').click();
+  };
+
+  fillOrderFields(orderName, userCountry, creditCard, month, city, year) {
+    cy.get('#name').type(orderName);
+    cy.get('#country').type(userCountry);
+    cy.get('#card').type(creditCard);
+    cy.get('#month').type(month);
+    cy.get('#city').type(city);
+    cy.get('#year').type(year);
+  };
+
+  confirmOrder() {
+    cy.contains('.btn', 'Purchase').click();
+  };
```

Comment:
There's an unnecessary semicolon at the end of the method definition. In JavaScript, semicolons are not required after method definitions within a class.